libs/product/testing/src/helpers/mock-configurable-product-facade.ts

```diff
@@ -0,0 +1,17 @@
+import { BehaviorSubject } from 'rxjs';
+import { Dictionary } from '@ngrx/entity';
+
+import { DaffConfigurableProductFacadeInterface } from '@daffodil/product';
+
+export class MockDaffConfigurableProductFacade implements DaffConfigurableProductFacadeInterface {
+	getAppliedAttributes(id: string): BehaviorSubject<Dictionary<string>> {
+		return new BehaviorSubject(null);
```

Comment:
I think this should be a `BehaviorSubject({})` to better match the return type.

Reply:
changed